.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why is this addition needed?

Reply:
Used that the dev-wf-ui can be imported as module in the designer
![image](https://github.com/axonivy/dev-workflow-ui/assets/42733123/8ec6dc32-ac1b-4d2c-8fe7-5acbbd945ea2)